.github/workflows/build-container.yml

```diff
@@ -41,13 +41,18 @@ jobs:
         SERIES: ${{ matrix.series }}
         TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
       run: |
-        IMAGE_REPO="${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}"
-        tags=( $IMAGE_REPO:$SERIES-el$DVER-$REPO{,-$TIMESTAMP} )
-        if [[ $SERIES-el$DVER-$REPO = 3.5-el7-testing ]]; then
-            tags+=( $IMAGE_REPO:{fresh,$TIMESTAMP} )
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tag_list=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          tag_list+=( $registry/$docker_repo:$SERIES-el$DVER-$REPO{,-$TIMESTAMP} )
+          if [[ $SERIES-el$DVER-$REPO = 3.5-el7-testing ]]; then
+            tags+=( $registry/$docker_repo:{fresh,$TIMESTAMP} )
         fi
-        tag_list=$(IFS=,; echo "${tags[*]}")
-        echo "::set-output name=taglist::${tag_list}"
+        done
+        # This causes the tag_list array to be comma-separated below,
+        # which is required for build-push-action
+        IFS=,
+        echo "::set-output name=taglist::${tag_list[*]}"
```

Comment:
Since the old lines are equivalent to the new lines, the best thing to do in Git would just be to keep the old lines and add the comment above them. This is because you can use `git blame` on a file to show which commit each line corresponds to and combined with checking out the file at the referenced commit + repeating the process, you can see the evolution of a specific piece of code in a file over time! So you want to help future developers by reducing the number of hops (i.e., commits) that they have to traverse.

CC @yongleyuan ^^ an important version control lesson